bazel/BUILD

```diff
@@ -0,0 +1,14 @@
+config_setting(
+    name = "wasm_runtime_wavm",
+    values = {"define": "wasm_runtime=wavm"},
```

Comment:
I think `runtime=XXX` should be self-explanatory in this context, adding `wasm` everywhere adds more noise than value.